src/sections/Projects/Sistent/components/card/code.js

```diff
@@ -0,0 +1,259 @@
+import React from "react";
+import { navigate } from "gatsby";
+import { useLocation } from "@reach/router";
+
+import { SistentThemeProvider, Card, CardActions, CardContent, CardMedia , Typography, Button } from "@layer5/sistent";
+import { CodeBlock } from "../button/code-block";
+import { SistentLayout } from "../../sistent-layout";
+
+import MesheryLogo from "../../../../../assets/images/meshery/meshery-logo-light-text-side.webp";
+import KanvasLogo from "../../../../../assets/images/kanvas/icon-only/kanvas-icon-color.png";
+
+import TabButton from "../../../../../reusecore/Button";
+import { useStyledDarkMode } from "../../../../../theme/app/useStyledDarkMode";
+
+const codes = [
+  `const cardOutlined = (
```

Comment:
Rather than creating a new component, you can directly pass as a child component to Card. So that I will be easier to copy for developers in the future.

Reply:
I made it this way to keep the code modular and abstract to make it more reusable. Do you still suggest I add this as a child component? @Vidit-Kushwaha